src/main/java/org/cactoos/http/HtHead.java

```diff
@@ -50,12 +50,18 @@
      */
     private final Input response;
 
+    /**
+     * Header end.
+     */
+    private final byte[] end;
```

Comment:
@krzyk why have this instance attribute instead of just using it in `findTail`?

Reply:
@llorllale because in that case findTail would need to be static and we don't want statics, right?

Reply:
@krzyk I don't understand; why does declaring this byte array inside `findEnd` require making the method static?